packages/components/src/Actions/ActionDropdown/ActionDropdown.snapshot.test.js

```diff
@@ -82,7 +82,7 @@ describe('ActionDropdown', () => {
 		const wrapper = mount(<ActionDropdown {...props} />).find('DropdownButton');
```

Comment:
`.find('DropdownButton')` always need?

Reply:
The test seems to check the dropdown button only. So querying the button avoid having the menu too

Reply:
Good for me